idemixmsp/msp_config.proto

```diff
@@ -0,0 +1,53 @@
+// IdemixMSPConfig collects all the configuration information for
+// an Idemix MSP.
+
+
+// Copyright the Hyperledger Fabric contributors. All rights reserved.
+//
+// SPDX-License-Identifier: Apache-2.0
+
+syntax = "proto3";
+option go_package = "github.com/IBM/idemix/idemixmsp";
+
+package idemixmsp;
+
+message IdemixMSPConfig {
+    // Name holds the identifier of the MSP
+    string name = 1;
+    
+    // ipk represents the (serialized) issuer public key
+    bytes ipk = 2;
+    
+    // signer may contain crypto material to configure a default signer
+    IdemixMSPSignerConfig signer = 3;
+    
+    // revocation_pk is the public key used for revocation of credentials
+    bytes revocation_pk = 4;
+    
+    // epoch represents the current epoch (time interval) used for revocation
+    int64 epoch = 5;
+    
+    // Which Elliptic Curve should be used
```

Comment:
the comments seem to start with the name of the field to follow the go convention